client/components/search/style.scss

```diff
@@ -158,6 +158,8 @@
 	}
 
 	.search__input-fade {
+		display: flex;
+		align-items: center;
```

Comment:
Thanks for jumping in on this one, @DustinHaefele!

This all works for me. Strange that we even have to create a work around for this bug. I can't find any reports of it elsewhere.

It looks like it's been fixed in Chromium (Build 78), though I have no clue how.